sdk/eventhub/Azure.Messaging.EventHubs/stress/templates/deploy-job.yaml

```diff
@@ -1,14 +1,43 @@
-{{- include "stress-test-addons.deploy-job-template.from-pod" (list . "stress.event-hub") -}}
-{{- define "stress.event-hub" -}}
-metadata:
-  labels:
-    testName: "net-eh-stress-{{ .Scenario }}"
-    testInstance: "eventhubs-{{.Scenario }}-{{ .Release.Name }}-{{ .Release.Revision }}"
+{{- include "stress.deploy-job-template.from-job" (list . "stress.deploy-example") -}}
+{{- define "stress.deploy-example" -}}
 spec:
-  containers:
-    - name: net-eh-stress
-      image: {{ .Values.image }}
-      imagePullPolicy: Always
-      command: ['dotnet', 'Azure.Messaging.EventHubs.Stress.dll', '--tests', {{ .Scenario }}]
-      {{- include "stress-test-addons.container-env" . | nindent 6 }}
+  {{ if eq .Stress.Scenario "EventProd" }}
+  completions: 2
+  parallelism: 2
+  {{ end }}
+  {{ if eq .Stress.Scenario "BuffProd" }}
+  completions: 2
+  parallelism: 2
+  {{ end }}
+  {{ if eq .Stress.Scenario "Processor" }}
+  completions: 2
+  parallelism: 2
+  {{ end }}
+  completionMode: Indexed
+  template:
+    metadata:
+      labels:
+        testName: "net-eh-stress-{{ lower .Stress.Scenario }}"
+        testInstance: "eventhubs-{{ lower .Stress.Scenario }}-{{ .Release.Name }}-{{ .Release.Revision }}"
+    spec:
+      containers:
+        - name: role
+          image: {{ .Values.image }}
+          command: ['dotnet', "Azure.Messaging.EventHubs.Stress.dll", "--test", "{{ .Stress.Scenario }}", "--role", "$(JOB_COMPLETION_INDEX)"]
```

Comment:
How will this fail if `JOB_COMPLETION_INDEX` is empty? e.g. if none of the above `if` statements resolve to true.

Reply:
If none of the if statements resolve to true then it will just set the JOB_COMPLETION_INDEX to 1, because `completionMode` is set to `indexed` outside of the if statements, and completions/parallelism default to 1.

Reply:
Ahh, I didn't know that. Makes sense.